src/renderer/component/expandable/view.jsx

```diff
@@ -48,6 +48,7 @@ export default class Expandable extends PureComponent<Props, State> {
         </div>
         <Button
           button="link"
+          className="expandable__button"
```

Comment:
Does a regular `link` not work? If not this should be a button class `btn--expandable`, but it should just use a regular link button